alohomora/req.py

```diff
@@ -282,9 +283,13 @@ def login_two_factor(self, response_1fa):
 
     def _get_form_action(self, soup):
         LOG.debug('Looking for the form action')
-        tag = soup.find('form')
-        LOG.debug('Found form action %s', tag['action'])
-        return tag['action']
+        form = soup.find('form')
+        if form is None:
+            print('Expected form not found, please make sure Duo is set up properly.')
```

Comment:
Could you update this to use `alohomora.die('str')` rather than printing and running `sys.exit()` directly?